py3-wheel.yaml

```diff
@@ -27,11 +33,36 @@ pipeline:
       tag: ${{package.version}}
       expected-commit: fa33dfd01fd665c1fd90097563b34bce4b5527ef
 
-  - runs: |
-      python3 -m gpep517 build-wheel --wheel-dir dist --output-fd 1
-      python3 -m installer -d "${{targets.destdir}}" dist/wheel-${{package.version}}-py3-none-any.whl
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package}} installed for python${{range.key}}
+    pipeline:
+      - uses: py/pip-build-install-bootstrap
+        with:
+          python: python${{range.key}}
+
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}-bin
+    description: Executable binaries for ${{vars.pypi-package}} installed for python${{range.key}}
+    dependencies:
+      runtime:
+        - py${{range.key}}-${{vars.pypi-package}}
+      provides:
+        - py3-${{vars.pypi-package}}
+      provider-priority: ${{range.value}}
```

Comment:
@smoser note package names above. Let me know if it is acceptable or weird.

Reply:
other option is to just not ship them at all unless/until they're needed.  
right now something has to explicitly depend / install py3.xx-wheels-bin . just do it later?